### ISSUE TITLE
HD issue x2

### ISSUE THREAD

**Issue by undonecroissant** [Delta2Force/MCVmComputers#151]
I have a hd that is saying I don't have enough space (which isn't accurate). When I click on the (reformat?) hard drive, I get brought back to the launcher screen. Please advise. I am running 1.16.3. Let me know if you need more information. I also already deleted the setting in the VB application and in the VM folder. Thank you.

![Screen Shot 2020-12-12 at 6 36 57 PM](https://user-images.githubusercontent.com/73410432/102000268-5058c080-3cab-11eb-823a-6dd507ae1638.png)

[crash-2020-12-12_18.18.23-client.txt](https://github.com/Delta2Force/MCVmComputers/files/5684011/crash-2020-12-12_18.18.23-client.txt)

**Comment by Busaruba2011** [comment]
Should I need to know, it says you have macOS 10.15.7. That's Catalina innit?
Try VirtualBox 6.1.14. That's what I use. I do it on Windows 10, don't have much experience with macOS other than in the Apple Store and school computers 😉

**Comment by Delta2Force** [comment]
Does it say "not enough space" no matter what you type in it?

**Comment by undonecroissant** [comment]
@Delta2Force Yes, it says that no matter what number I put in. I was able to find the folder for the HD and just delete what was is it. @Busaruba2011 I'll try that and report back. Thank you.

**Comment by undonecroissant** [comment]
I am using VB 6.1.14. That wasn't working. I kept getting a  
com.apple.TCC                 	
0x00007fff64a0b59f __CRASHING_DUE_TO_PRIVACY_VIOLATION__ + 163 e
error. If I sudo VirtualBox though, the system seems to work. I don't understand why that makes a difference, but my machine is now working.

**Comment by Delta2Force** [comment]
Huh.

**Comment by Busaruba2011** [comment]
> I am using VB 6.1.14. That wasn't working. I kept getting a  
> com.apple.TCC                 	
> 0x00007fff64a0b59f __CRASHING_DUE_TO_PRIVACY_VIOLATION__ + 163 e
> error. If I sudo VirtualBox though, the system seems to work. I don't understand why that makes a difference, but my machine is now work

@undonecroissant i found this https://developer.apple.com/forums/thread/659732
Someone said in a terminal, type 'sudo virtualbox'

**Comment by CoccodrillooXDS** [comment]
> I am using VB 6.1.14. That wasn't working. I kept getting a
> com.apple.TCC
> 0x00007fff64a0b59f **CRASHING_DUE_TO_PRIVACY_VIOLATION** + 163 e
> error. If I sudo VirtualBox though, the system seems to work. I don't understand why that makes a difference, but my machine is now working.

Try you start manually the VM from VirtualBox?

**Comment by CoccodrillooXDS** [comment]
> I am using VB 6.1.14. That wasn't working. I kept getting a
> com.apple.TCC
> 0x00007fff64a0b59f **CRASHING_DUE_TO_PRIVACY_VIOLATION** + 163 e
> error. If I sudo VirtualBox though, the system seems to work. I don't understand why that makes a difference, but my machine is now working.

Try starting the VM from VirtualBox (and allow privacy options if required) and see if it works

**Comment by Johnathan236** [comment]
Hello i have the same problem but Hard drive cannot get in into the pc Thank you
